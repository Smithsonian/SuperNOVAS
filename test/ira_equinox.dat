
2450545.000000 TEST       S2 O0 A1:     0.002338     0.002358

2450545.000000 TEST       S2 O1 A1:     0.002338     0.002358

2450545.000000 TEST       S2 O2 A1:     0.002338     0.002358

2450545.000000 SUN        S0 O0 A1:     0.002338     0.002358

2450545.000000 SUN        S0 O1 A1:     0.002338     0.002358

2450545.000000 SUN        S0 O2 A1:     0.002338     0.002358

2451545.000000 TEST       S2 O0 A1:    -0.000000     0.000236

2451545.000000 TEST       S2 O1 A1:    -0.000000     0.000236

2451545.000000 TEST       S2 O2 A1:    -0.000000     0.000236

2451545.000000 SUN        S0 O0 A1:    -0.000000     0.000236

2451545.000000 SUN        S0 O1 A1:    -0.000000     0.000236

2451545.000000 SUN        S0 O2 A1:    -0.000000     0.000236

2452545.000000 TEST       S2 O0 A1:    -0.002339    -0.002052

2452545.000000 TEST       S2 O1 A1:    -0.002339    -0.002052

2452545.000000 TEST       S2 O2 A1:    -0.002339    -0.002052

2452545.000000 SUN        S0 O0 A1:    -0.002339    -0.002052

2452545.000000 SUN        S0 O1 A1:    -0.002339    -0.002052

2452545.000000 SUN        S0 O2 A1:    -0.002339    -0.002052
