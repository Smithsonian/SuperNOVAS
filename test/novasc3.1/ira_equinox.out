  -10000.000 A0:  0.023381867  0.023103017
       0.000 A0: -0.000000269  0.000236403
   10000.000 A0: -0.023386268 -0.023566385
   10000.000 A0: -0.023386268 -0.023566385
   10000.010 A0: -0.023386291 -0.023566416
  -10000.000 A1:  0.023382     0.023103   
       0.000 A1: -0.000000     0.000236   
   10000.000 A1: -0.023386    -0.023566   
   10000.000 A1: -0.023386    -0.023566   
   10000.010 A1: -0.023386    -0.023566   
